lib/cmd/fh3/notifications/list.js

```diff
@@ -0,0 +1,50 @@
+/* globals i18n */
+var common = require('../../../common.js');
+var fhreq = require("../../../utils/request");
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('List notifications of a RHMAP App'),
+  'examples' : [{
+    cmd : 'fhc notifications list --app=<app> --env=<environment> --audit=<true>',
+    desc : i18n._('List notifications of the <app>')}],
+  'demand' : ['app', 'env'],
+  'alias' : {
+    'app':'a',
+    'env':'e',
+    'audit':'au',
+    'json': 'j',
+    0 : 'app',
+    1 : 'env',
+    2 : 'audit',
```

Comment:
A position is an option for work with it. Following few examples

It worked as:

```
fhc notifications list <app> <environment> <true>
fhc notifications list --app=<app> --env=<environment> --audit=<true> --json
fhc notifications list --a=<app> --e=<environment> --au=<true> --j=<true>
fhc notifications list <app> <environment> <true> <true>
```